src/cubic_loader/qlik/rds_utils.py

```diff
@@ -41,10 +41,10 @@ def qlik_type_to_pg(qlik_type: str, scale: int) -> str:
         return_type = "INTEGER"
     elif "INT4" in qlik_type:
         return_type = "BIGINT"
-    elif "NUMERIC" in qlik_type and scale == 0:
```

Comment:
This is a change that has already been implemented on `odin` there are sometimes numeric types with scale=0 that can overflow the `BIGINT` type